packages/react-dom-bindings/src/client/ReactFiberConfigDOM.js

```diff
@@ -2331,6 +2332,23 @@ function removeEventListenerFromChild(
   return false;
 }
 // $FlowFixMe[prop-missing]
+FragmentInstance.prototype.dispatchEvent = function (
+  this: FragmentInstanceType,
+  event: Event,
+): boolean {
+  const parentHostInstance = getFragmentParentHostInstance(this._fragmentFiber);
+  if (parentHostInstance === null) {
+    if (__DEV__) {
+      console.error(
+        'You are attempting to dispatch an event on a disconnected ' +
+          'FragmentInstance. No event was dispatched.',
+      );
```

Comment:
I wonder if this is necessary. It might backfire if you don't know, now you have to check with something like `getRootNode()` instead.

Reply:
Yeah probably more annoying than anything. Updated for it to just exit silently.